jekyll/_cci2/server/operator/troubleshooting-and-support.adoc

```diff
@@ -15,6 +15,34 @@ This document describes an initial set of troubleshooting steps to take if you a
 
 toc::[]
 
+[#generate-support-bundle]
+== Generate Support Bundle
+A support bundle is used by CircleCI engineers to diagnose and fix any issues you are experiencing. They are typically requested when you open a ticket.
+
+To generate a support bundle, You need to follow below steps - 
+
+=== Prerequisites
+. First, make sure circleci-server is deployed and you have access to the cluster/namespace through kubectl.
+
+. Next, install link:https://krew.sigs.k8s.io/docs/user-guide/setup/install/[krew].
+
+. Install preflight and support bundle to your local development machine.
```

Comment:
I'm not clear on this step - what's 'preflight and support bundle' in this context?